ftplugin/php.vim

```diff
@@ -40,6 +40,12 @@ let g:phpactorInputListStrategy = get(g:, 'phpactorInputListStrategy', 'phpactor
 " to that window instead of switching buffers.  The default is false.
 let g:phpactorUseOpenWindows = get(g:, 'phpactorUseOpenWindows', v:false)
 
+" The list of files that determine workspace root directory.
```

Comment:
maybe leave empty to assume the CWD or similar?

Reply:
> Add .phpactor.json and .phpactor.yml here?
> maybe leave empty to assume the CWD or similar?

If I understand, the one above sentences contradicts with each other. 

We would set the default empty and show `['composer.json', '.git', '.phpactor.json', '.phpactor.yml']` as an example in the docblock.